tex/chProofs.tex

```diff
@@ -370,15 +396,17 @@ \subsection{Conditional statements}
 same as the one we have used in chapter~\ref{ch:prog} in order to
 represent function types. This is no accident, but we postpone further
 comments on the meaning of this arrow to section~\ref{sec:chi}. For
-now let us only stress that \C{->} associates on the right and that
+now let us only stress that \C{->} associates on the right, and
```

Comment:
"associates to to the right(left)" seems like an idiom in Computer Science English. The usual context for the idiom is function types or application terms. E.g. it is used in the popular  [TAPL](https://www.cis.upenn.edu/~bcpierce/tapl/) book.

Wiki uses "from" it like so: "right-associative (meaning the operations are grouped from the right)".